dev-client/components/Login.tsx

```diff
@@ -0,0 +1,36 @@
+import {Button, VStack} from 'native-base';
+import {useCallback} from 'react';
+import {authorize} from 'react-native-app-auth';
+
+import Config from 'react-native-config';
+import {exchangeToken} from '../src/auth';
+import {useLoginDispatch} from '../src/context/LoginContext';
+
+// https://github.com/FormidableLabs/react-native-app-auth/blob/main/docs/config-examples/google.md
+const googleConfig = {
+  issuer: 'https://accounts.google.com',
+  clientId: `${Config.GOOGLE_OAUTH_APP_GUID}.apps.googleusercontent.com`,
+  redirectUrl: 'com.devclient:/oauth2redirect',
```

Comment:
question: where do we configure how we handle this redirect URL?

Reply:
answer: the scheme name is handled by the changes to [build.gradle](https://github.com/techmatters/terraso-mobile-client/blob/50a3cd16e0042962b0f0c14c407b4bee94e64289/dev-client/android/app/build.gradle#L104). I need to update this actually so that the package name is `org.terraso.landpks`, I will add a task to do so. Ideally this would be at least a constant value for when we add other OAuth providers as well.

As to the route path: I am just following the react-native-app-auth docs I linked in the comment above. I checked the [Google docs](https://developers.google.com/identity/protocols/oauth2/native-app#step-2:-send-a-request-to-googles-oauth-2.0-server) as well, and it says the path part is optional, so idk. I might try and see if it works without the `oauth2redirect`, might not.